code/src/WijmoProvider/Columns/DateTimeColumn.ts

```diff
@@ -48,7 +48,9 @@ namespace WijmoProvider.Column {
 
         public build(): void {
             this.editorConfig.timeFormat = 'HH:mm';
-            this.config.format = `${GridAPI.dateFormat} ${this.editorConfig.timeFormat}`;
+            this.config.format =
+                this.editorConfig.format ||
+                `${GridAPI.dateFormat} ${this.editorConfig.timeFormat}`;
```

Comment:
Is this the only way to get the dateFormat? We should avoid accessing the GridAPI on the wijmoProvider

Reply:
Agree, we should preferably always use the config, now that we have it. Meaning that when we're loading the configs we should set the value if they come empty, and from them on, the configs are "master".